OpenRA.Mods.Common/Traits/Replaceable.cs

```diff
@@ -20,8 +20,8 @@ namespace OpenRA.Mods.Common.Traits
 	public class ReplaceableInfo : ITraitInfo
 	{
 		[FieldLoader.Require]
-		[Desc("Identifyer for this replaceable Actor (matched against Replaces in Replacement).")]
-		public readonly string Type = null;
+		[Desc("Replacement types this Relpaceable actor accepts.")]
+		public readonly string[] Replacements = null;
```

Comment:
Sorry, I meant to keep this as a list of arbitrary types, rather than actor names, but just to change it to an array that accepts multiple!